drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/handler/AbstractBinaryHandler.java

```diff
@@ -185,7 +185,7 @@ protected File downloadAndPrepare(File targetDir, URL from) throws Exception {
         File downloaded = Downloader.download(targetDir, from);
         File extraction = BinaryFilesUtils.extract(downloaded);
         File[] files = extraction.listFiles(file -> file.isFile());
```

Comment:
I don't think that it is necessary. From my point of view, it is something that we should have done a long time ago, I was only waiting till Selenium changes its support of java version first. 
For those who would be really interested in it, it is available in the pom.xml file :-)